managed_consumer.go

```diff
@@ -339,3 +339,86 @@ func (m *ManagedConsumer) manage() {
 		m.set(consumer)
 	}
 }
+
+// RedeliverUnacknowledged requests redeliver_unacknowledged_messages request
+// for all messages that have not been acked.
```

Comment:
The comment seems to have a typo.

Reply:
Hi Adam! Just bumping this review so it doesn't hang too long.